YIF_Backend/Controllers/InstitutionOfEducationModeratorController.cs

```diff
@@ -78,5 +78,24 @@ public async Task<IActionResult> UpdateSpecialtyDescription([FromBody] Specialty
             var result = await _ioEModeratorService.UpdateSpecialtyDescription(specialtyDescriptionUpdateApiModel);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Get specialty description in IoE.
+        /// </summary>
+        /// <response code="200">Get full description of specialty in IoE</response>
+        /// <response code="400">If id is not valid.</response>
+        /// <response code="401">If user is unauthorized, token is bad/expired</response>
+        /// <response code="403">If user is not institution of education moderator</response>
+        /// <response code="404">Specialty not found</response>
+        [HttpGet("Specialty/Description/Get/{specialtyId}")]
+        [ProducesResponseType(typeof(SpecialtyToInstitutionOfEducationResponseApiModel), 200)]
+        [ProducesResponseType(typeof(DescriptionResponseApiModel), 404)]
```

Comment:
Same question: Is it DescriptionResponseApiModel?

Reply:
Yes.